cmd/harmony/main.go

```diff
@@ -877,41 +872,6 @@ func setupSyncService(node *node.Node, host p2p.Host, hc harmonyconfig.HarmonyCo
 		blockchains = append(blockchains, node.EpochChain())
 	}
 
-	dConfig := downloader.Config{
-		ServerOnly:   !hc.Sync.Downloader,
-		Network:      nodeconfig.NetworkType(hc.Network.NetworkType),
-		Concurrency:  hc.Sync.Concurrency,
-		MinStreams:   hc.Sync.MinPeers,
-		InitStreams:  hc.Sync.InitStreams,
-		SmSoftLowCap: hc.Sync.DiscSoftLowCap,
-		SmHardLowCap: hc.Sync.DiscHardLowCap,
-		SmHiCap:      hc.Sync.DiscHighCap,
-		SmDiscBatch:  hc.Sync.DiscBatch,
```

Comment:
if those config are no longer used, we also need to update the default config for each network

Reply:
We use these configs in staged stream sync as well.